### PR TITLE
[Duplicate] [bad branch name] upgrade library: added has_package check

### PR DESCRIPTION
**Pull request by fernflower**
Added method to the signed/unsigned rpms models to
check for the given package presence.
This should reduce a bit code duplication in any other actors
that check that the given package is present in the system.

### CODE REVIEW

File: repos/system_upgrade/el7toel8/libraries/rpms.py [hunk 1]
```diff
@@ -16,3 +16,12 @@ def get_installed_rpms():
                                                                                               RC=err.exit_code)
         stdlib.api.current_logger().error(error)
         return []
+
+
+def create_lookup(model, field, key, context):
+    return {getattr(obj, key) for obj in getattr(next((m for m in context.consume(model)), model()), field)}
```
Comment (examon) [inline repos/system_upgrade/el7toel8/libraries/rpms.py]: Please, would it be possible to rewrite this oneliner into more simpler constructs? I get what you want to do here but it hurts my brain too much to unpack the logic behind this (or add docstring explaining what is going on in simple terms). Thanks

Comment (fernflower) [reply repos/system_upgrade/el7toel8/libraries/rpms.py]: Sure, sorry for not putting wip label, it was kind of not review ready

**Comment by vinzenz** [comment]
@fernflower Tests are still failing though

**Comment by fernflower** [comment]
leapp-ci build

**Comment by fernflower** [comment]
Closing in favor of https://github.com/oamg/leapp-repository/pull/225

**Pull request closed without merging**